docs/book/src/user/quick-start.md

```diff
@@ -647,7 +647,7 @@ kubeadmconfigtemplate.bootstrap.cluster.x-k8s.io/capi-quickstart-md-0 created
 The cluster will now start provisioning. You can check status with:
 
 ```bash
-kubectl get cluster --all-namespaces
+kubectl get cluster
```

Comment:
We only have one cluster which we deploy in the default namespace so --all-namespaces is not required here. We also have other kubectl commands in the guide which assume the default namespace (like delete cluster below)